ProjectLighthouse/Types/Filter/FilterLocation.cs

```diff
@@ -0,0 +1,13 @@
+namespace LBPUnion.ProjectLighthouse.Types.Filter;
+
+public enum Location
```

Comment:
```suggestion
public enum FilterLocation
```
I think this should have a more specific name otherwise it could be confused with the Location used for users and levels.